RInAppMessaging/Classes/RInAppMessaging.swift

```diff
@@ -1,3 +1,9 @@
+/// Protocol for optional delagate
+@objc public protocol RInAppMessagingDelegate: AnyObject {
+    /// Method called only for campaigns with context just before displaying its message
+    func inAppMessagingShouldShowCampaignsWithContexts(contexts: [String], campaignTitle: String) -> Bool
```

Comment:
`Campaigns` or `Campaign` ?
Cos there's `campaignTitle` which suggests singular campaign.

Reply:
Why are there multiple contexts? To provide flexibility?

Reply:
yes. Single campaign, multiple contexts